### PR TITLE
HDDS-2141. Missing total number of operations

### PR DESCRIPTION
**Pull request by adoroszlai**
## What changes were proposed in this pull request?

Sum request counts and use it as fallback in case explicit "Ops" metric is not available.

https://issues.apache.org/jira/browse/HDDS-2141

## How was this patch tested?

Ran `ozonefs` robot test, checked OM metrics page.

### CODE REVIEW

File: hadoop-ozone/ozone-manager/src/main/resources/webapps/ozoneManager/ozoneManager.js [hunk 1]
```diff
@@ -87,6 +88,7 @@
                                 if (name == "Ops") {
                                     groupedMetrics.nums[type].ops = metrics[key]
                                 } else {
+                                    groupedMetrics.nums[type].total += metrics[key]
```
Comment (hadoop-yetus) [inline hadoop-ozone/ozone-manager/src/main/resources/webapps/ozoneManager/ozoneManager.js]: jshint:84:W033:Missing semicolon.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 164 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| -1 | mvninstall | 35 | hadoop-ozone in trunk failed. |
| -1 | compile | 25 | hadoop-ozone in trunk failed. |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 1088 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 206 | trunk passed |
||| _ Patch Compile Tests _ |
| -1 | mvninstall | 41 | hadoop-ozone in the patch failed. |
| -1 | jshint | 106 | The patch generated 1393 new + 2737 unchanged - 0 fixed = 4130 total (was 2737) |
| -1 | compile | 26 | hadoop-ozone in the patch failed. |
| -1 | javac | 26 | hadoop-ozone in the patch failed. |
| +1 | mvnsite | 0 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 802 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 265 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 227 | hadoop-hdds in the patch failed. |
| -1 | unit | 108 | hadoop-ozone in the patch failed. |
| +1 | asflicense | 114 | The patch does not generate ASF License warnings. |
| | | 3663 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.ozone.lock.TestLockManager |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1462 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient jshint |
| uname | Linux 03d06927b330 4.15.0-54-generic #58-Ubuntu SMP Mon Jun 24 10:55:24 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / c474e24 |
| Default Java | 1.8.0_222 |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/1/artifact/out/branch-mvninstall-hadoop-ozone.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/1/artifact/out/branch-compile-hadoop-ozone.txt |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/1/artifact/out/patch-mvninstall-hadoop-ozone.txt |
| jshint | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/1/artifact/out/diff-patch-jshint.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/1/artifact/out/patch-compile-hadoop-ozone.txt |
| javac | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/1/artifact/out/patch-compile-hadoop-ozone.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/1/artifact/out/patch-unit-hadoop-hdds.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/1/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/1/testReport/ |
| Max. process+thread count | 306 (vs. ulimit of 5500) |
| modules | C: hadoop-ozone/ozone-manager U: hadoop-ozone/ozone-manager |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/1/console |
| versions | git=2.7.4 maven=3.3.9 jshint=2.10.2 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 40 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| -1 | mvninstall | 31 | hadoop-ozone in trunk failed. |
| -1 | compile | 20 | hadoop-ozone in trunk failed. |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 1004 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 187 | trunk passed |
||| _ Patch Compile Tests _ |
| -1 | mvninstall | 34 | hadoop-ozone in the patch failed. |
| -1 | jshint | 93 | The patch generated 1392 new + 2737 unchanged - 0 fixed = 4129 total (was 2737) |
| -1 | compile | 26 | hadoop-ozone in the patch failed. |
| -1 | javac | 27 | hadoop-ozone in the patch failed. |
| +1 | mvnsite | 0 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 802 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 176 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 262 | hadoop-hdds in the patch passed. |
| -1 | unit | 31 | hadoop-ozone in the patch failed. |
| +1 | asflicense | 36 | The patch does not generate ASF License warnings. |
| | | 3123 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1462 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient jshint |
| uname | Linux c8863d1e4b09 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / e97f0f1 |
| Default Java | 1.8.0_222 |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/2/artifact/out/branch-mvninstall-hadoop-ozone.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/2/artifact/out/branch-compile-hadoop-ozone.txt |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/2/artifact/out/patch-mvninstall-hadoop-ozone.txt |
| jshint | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/2/artifact/out/diff-patch-jshint.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/2/artifact/out/patch-compile-hadoop-ozone.txt |
| javac | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/2/artifact/out/patch-compile-hadoop-ozone.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/2/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/2/testReport/ |
| Max. process+thread count | 440 (vs. ulimit of 5500) |
| modules | C: hadoop-ozone/ozone-manager U: hadoop-ozone/ozone-manager |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1462/2/console |
| versions | git=2.7.4 maven=3.3.9 jshint=2.10.2 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by adoroszlai** [comment]
Thanks @elek for reviewing and merging it.

**Pull request closed without merging**